test/docs/jdk/javadoc/doccheck/doccheckutils/checkers/LinkChecker.java

```diff
@@ -220,20 +220,14 @@ private void reportMissingFile(Path file) {
 
     @Override
     public boolean isOK() {
-        return duplicateIds == 0
-                && missingIds == 0
-                && missingFiles == 0
-                && badSchemes == 0;
+        return log.noErrors() && (missingFiles == 0);
```

Comment:
In method `reportMissingFile` where `missingFiles` is increased an error is also logged, so I'm wondering is the `missingFiles` check needed here?